smqtk_descriptors/impls/descriptor_generator/prepopulated.py

```diff
@@ -0,0 +1,124 @@
+# copy of pytorch
+import abc
+import copy
+import itertools
+import logging
+from typing import (
+    Any, Callable, Dict, Iterable, Iterator, Literal, Mapping,
+    Optional, Sequence, Set, Type, TypeVar, Union
+)
+
+import numpy as np
+
+from smqtk_dataprovider import DataElement
+from smqtk_descriptors import DescriptorGenerator
+from smqtk_image_io import ImageReader
+from smqtk_core.configuration import (
+    from_config_dict,
+    make_default_config,
+    to_config_dict,
+)
+from smqtk_descriptors.utils.parallel import parallel_map
+
+LOG = logging.getLogger(__name__)
+
+__all__ = [
+    "PrePopulatedDescriptorGenerator"
+
+]
+T = TypeVar("T", bound="PrePopulatedDescriptorGenerator")
+
+
+
+
+class PrePopulatedDescriptorGenerator (DescriptorGenerator):
+
+
+    @classmethod
+    def is_usable(cls) -> bool:
+        return True
+
+    @classmethod
+    def get_default_config(cls) -> Dict[str, Any]:
+        c = super().get_default_config()
+        return c
+
+    @classmethod
+    def from_config(
+        cls: Type[T],
+        config_dict: Dict,
+        merge_default: bool = True
+    ) -> T:
+        # Copy config to prevent input modification
+        config_dict = copy.deepcopy(config_dict)
+        return super().from_config(config_dict, merge_default)
+
+    def __init__(
+        self,
+    ):
+        super().__init__()
+
+    @abc.abstractmethod # might be internal and never called
+    def _load_module(self) -> "torch.nn.Module":
+        """
+        :return: Load and return the module.
+        """
```

Comment:
This doesn't seem to be necessary form this class and is an unnecessary copy-paste from the pytorch interface you based this on. Remove this and this implementation should be discoverable again.

Reply:
This abstract class has been removed.